pygsheets/authorization.py

```diff
@@ -112,13 +112,10 @@ def authorize(client_secret='client_secret.json',
         if key in ['outh_file', 'outh_creds_store', 'service_file', 'credentials']:
             warnings.warn('The argument {} is deprecated. Use {} instead.'.format(key, _deprecated_keyword_mapping[key])
                           , category=DeprecationWarning)
-    client_secret = kwargs.get('outh_file', client_secret)
-    service_account_file = kwargs.get('service_file', service_account_file)
-    credentials_directory = kwargs.get('outh_creds_store', credentials_directory)
-    custom_credentials = kwargs.get('credentials', custom_credentials)
-
-    http = kwargs.get('http', None)
-    check = kwargs.get('check', True)
```

Comment:
Don't need to do anything explicit with these since `None` and `True` are already the default values in `Client` anyway